RELEASE/scripts/sl_ascend/sl_combat.ash

```diff
@@ -1949,6 +1949,13 @@ string sl_combatHandler(int round, string opp, string text)
 		if(canUse($skill[Blood Chains], false) && my_hp() > 3 * hp_cost($skill[Blood Chains]))
 			stunner = useSkill($skill[Blood Chains], false);
 		// intentionally not setting costMinor or costMajor since they don't cost mp...
+
+		// If we're in a form or something, a beehive is probably better than just attacking
+		if(attackMajor == "attack with weapon" && !have_skill($skill[Preternatural Strength]) && canUse($item[beehive]) && ($stat[moxie] != weapon_type(equipped_item($slot[Weapon]))))
+		{
+			attackMajor = useItem($item[beehive], false);
+			attackMinor = useItem($item[beehive], false);
```

Comment:
How thoroughly have you tested this behavior? I'm pretty sure `canUse($item[beehive])` will always return true unless you do `useItem($item[beehive], true)` (or just leave off the boolean entirely). And you can't very safely do that here because it's not guaranteed that `attackMajor` or `attackMinor` will be used... I should refactor it to have a boolean that tracks whether attackMajor/attackMinor needs to be added to the combat state, and do so if necessary.

Reply:
canUse() also checks whether you have the item, so it should be fine. My run hasn't been clobbered by failing to use the beehive and spamming attacks.

I'm OK with it using the beehive until the monster is dead. It does more damage than flailing around with a melee weapon.

I'm also OK with the beehive not actually getting used if attackMajor and attackMinor aren't used - since we're using the beehive as a reusable replacement attack, if we wouldn't otherwise attack, it's fine to not use the beehive I think.

Reply:
Oh - I'm passing in false because we can use the beehive as many times as we want - so I don't want it to be marked as used. It still damages, it just doesn't stagger.

Reply:
Oh, ok. I didn't realize the beehive was reusable in the same combat. Yep, that's not a problem then.